src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/ApiTests/AuthorizationApiTests.cs

```diff
@@ -0,0 +1,187 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+using Altinn.App.Common.Models;
+using Altinn.App.IntegrationTests;
+using Altinn.Platform.Register.Models;
+using App.IntegrationTests.Utils;
+using App.IntegrationTestsRef.Utils;
+
+using Newtonsoft.Json;
+using Xunit;
+
+namespace App.IntegrationTestsRef.ApiTests
+{
+    public class AuthorizationApiTests: IClassFixture<CustomWebApplicationFactory<Altinn.App.Startup>>
+    {
+        private readonly CustomWebApplicationFactory<Altinn.App.Startup> _factory;
+
+        public AuthorizationApiTests(CustomWebApplicationFactory<Altinn.App.Startup> factory)
+        {
+            _factory = factory;
+        }
+
+        [Fact]
+        public async Task GetCurrentParty_ReturnPartyID_OK()
+        {
+            string token = PrincipalUtil.GetToken(1337);
```

Comment:
The userId value and partyId value should be different. It is too easy to make mistakes and mix the two. The test can be verify that they are separate if the values are different.

Reply:
I guess there are other tests that verifies this to some degree, but they also involve the cookie value.

Reply:
We should do this is a wider change on partyid on all tests